engine/installation/linux/docker-ee/centos.md

```diff
@@ -38,11 +40,14 @@ Docker EE. On production systems, you must use `direct-lvm` mode, which requires
 one or more dedicated block devices. Fast storage such as solid-state media
 (SSD) is recommended.
 
-### Uninstall old versions
+#### Uninstall old versions (if present)
```

Comment:
We have never had anyone confused about this yet and I'm not sure it needs clarification.